config/SpecialAI.cfg

```diff
@@ -0,0 +1,214 @@
+##########################################################################################################
+
+jockeys {
+    # (0.0-1.0) Chance for a valid rider mob to actually get the rider AI. Default is 10% chance.
+    D:_rider_chance=0.1
+
+    # (Entity list) List of mobs that can be ridden on by normal-sized riders (normally non-aggressive mobs must have pathfinding AI enabled). Defaults to any kind of spider, pig, sheep, or cow.
+    S:mount_list=Spider,Pig,Sheep,Cow
+
+    # (Entity list) List of mobs that can be ridden on by small riders or normal-sized riders that are babies (normally non-aggressive mobs must have pathfinding AI enabled). Defaults to any kind of chicken.
+    S:mount_list_small=Chicken
+
+    # (Entity list) List of mobs that can ride normal-sized mounts (note that the entity must have pathfinding AI enabled). Defaults to any kind of zombie, skeleton, or creeper.
+    S:rider_list=Zombie,Skeleton,Creeper
+
+    # (Entity list) List of mobs that can only ride small mounts (note that the entity must have pathfinding AI enabled). Defaults to none.
+    S:rider_list_small=
+}
+
+
+##########################################################################################################
+# passive_griefing
+#--------------------------------------------------------------------------------------------------------#
+# Options to customize monsters' passive block griefing.
+##########################################################################################################
+
+passive_griefing {
+    # (True/false) If true, mobs will passively grief you while not doing anything else. Default is true.
+    B:_enabled=true
+
+    # (True/false) If true, mobs will passively flip switches, press buttons, etc. while not doing anything else (uses the same range, reach, etc. as griefing, but has separate entity and block lists). Default is true.
+    B:_fiddling_enabled=true
+
+    # (True/false) If true, block breaking AI will automatically target all light sources. Default is true.
+    B:break_lights=true
+
+    # (True/false) If true, a lound snapping sound will be played when greifing, which is audible regardless of distance. Default is false.
+    B:break_sound=false
+
+    # (0.0-INFINITY) Mobs' reach (from eye height) when griefing blocks. Player reach is about 4.5. Default is 3.5.
+    D:grief_range=3.5
+
+    # (Integer >= 0) The range at which mobs will search for blocks to grief horizontally. Default is 12 blocks.
+    I:grief_scan_range=12
+
+    # (Integer >= 0) The range at which mobs will search for blocks to grief vertically. Default is 6 blocks.
+    I:grief_scan_range_vertical=6
+
+    # (True/false) If true, griefed blocks will leave item drops. Default is true.
+    B:leave_drops=true
+
+    # (True/false) If true, creepers will be very mad about not having arms to break things with, and resort to what they know best... Default is true.
+    B:mad_creepers=true
```

Comment:
Set this false, creeper explosions damaging things will be turned off as well to not make terrain awful looking.